src/hooks/use-nonprofit.js

```diff
@@ -1,288 +1,278 @@
+        },
+      };
+
+      console.log("Authenticated user, adding token to request");
+      const data = await makeRequest({ config, authenticated: true });
+      onComplete(data);
+    } else {
+      if (ip === null || ip === undefined || ip === "") {
+        console.log("IP is null, cannot make request");
+        return null;
+      } else {
+        console.log("IP is not null, making request");
+      }
+
+      const config = {
+        url: `${apiNodeJsServerUrl}/api/public/nonprofit-application`,
+        method: "GET",
+        headers: {
+          Accept: "application/json",
+          "X-IP-Address": ip,
+        },
+      };
+      console.log("Unauthenticated user");
+      const data = await makeRequest({ config, authenticated: false });
+
+      console.log("Data from request: ", data);
+      onComplete(data);
+    }
 
-    const handle_get_nonprofit_applications = async ( ) => {        
-        const config = {
-            url: `${apiNodeJsServerUrl}/api/nonprofit-applications`,
-            method: "GET",
-            headers: {
-                "content-type": "application/json",
-            },
-        };
-
-        // Publically available, so authenticated: false here
-        const data = await makeRequest({ config, authenticated: false });
+    //return data;
+  };
 
-        if( user ){
-            setNonprofitApplications(data);
-        }        
-        
-        return data;
+  const handle_get_nonprofit_applications = async () => {
+    const config = {
+      url: `${apiNodeJsServerUrl}/api/nonprofit-applications`,
+      method: "GET",
+      headers: {
+        "content-type": "application/json",
+      },
     };
 
+    // Publically available, so authenticated: false here
+    const data = await makeRequest({ config, authenticated: false });
 
+    if (user) {
+      setNonprofitApplications(data);
+    }
 
-    const handle_new_npo_submission = async (name, email, npoName, description, website, slack_channel, checked, onComplete) => {
-        if (!user)
-            return null;
-
-        const config = {
-            url: `${apiServerUrl}/api/messages/npo`,
-            method: "POST",
-            headers: {
-                "Content-Type": "application/json",
-                "Accept": "application/json"
-            },
-            data: {
-                name: name,
-                email: email,
-                npoName: npoName,
-                description: description,
-                website: website,
-                slack_channel: slack_channel,
-                // We send just the ID to the backend, it will convert this to an object
-                // Ref: https://stackoverflow.com/a/59394211
-                problem_statements: checked.map(item => {
-                    return `${item}`
-                })
-            }
-        };
-
-        const data = await makeRequest({ config, authenticated: true });
-        onComplete(data.text);
-        return data;
+    return data;
```

Comment:
Hey @lucy-xfl  - I think that you formatted the code in your IDE? It's normally a good practice to not mix code formatting with a code change as it's really hard to tell what you changed, versus what the code formatter did.

Can you confirm this was formatted?

Reply:
Get it! Yes, I used prettier to format it. Do I need to recreate a branch and re-submit it again? I add some details in 32 line. Do you have any advice to make it more clear? I will take care of this detail next time.
<img width="730" alt="image" src="https://github.com/opportunity-hack/frontend-ohack.dev/assets/86309711/a0b6d6b9-fd82-4157-892c-32962c9f74f1">

Reply:
Sounds good - no need to do anything now, but generally this would have been two different commits, one for the formatting change and one for the code change/improvement/fix